Volume/Networking/Network.swift

```diff
@@ -87,3 +87,16 @@ struct GraphQLPublisher<Query: GraphQLQuery>: Publisher {
         subscriber.receive(subscription: subscription)
     }
 }
+
+class NetworkState: ObservableObject {
+    @Published var networkFailed = false
+
+    func handleNetworkFailure(_ completion: Subscribers.Completion<WrappedGraphQLError>) {
```

Comment:
This handles more than just failure, so is there a more descriptive name?

Reply:
I agree I need to change it.  However, I actually don't think it does more than just handle a failure (the function doesn't do too much).  I actually think it does something different (analyze completion and set networkFailed based on whether the completion gives an error).  I'm thinking about changing it to **determineState()**.  Do you think that is fine?